apps/app/platform.tf

```diff
@@ -0,0 +1,116 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+resource "azurerm_application_insights" "app" {
+  name                       = "ai-${local.app_id_truncated}-${var.naming_suffix}"
+  resource_group_name        = var.resource_group_name
+  location                   = var.location
+  workspace_id               = data.azurerm_log_analytics_workspace.core.id
+  application_type           = "web"
+  internet_ingestion_enabled = var.local_mode ? true : false
+}
+
+resource "azurerm_linux_web_app" "app" {
```

Comment:
We could do - thought that since at the moment with no vpn in place, do we know enough about what IPs testers will be using? Happy to add it if you've got a suggestion, or we can add in later?